tests/filecheck/transforms/convert-stencil-to-ll-mlir.mlir

```diff
@@ -309,20 +309,20 @@ builtin.module {
     "stencil.store"(%56, %50) {"lb" = #stencil.index<0>, "ub" = #stencil.index<64>} : (!stencil.temp<[0,64]xf64>, !stencil.field<[-4,68]xf64>) -> ()
     func.return
   }
-// CHECK:         func.func @stencil_buffer(%114 : memref<72xf64>, %115 : memref<72xf64>) {
-// CHECK-NEXT:      %116 = "memref.subview"(%115) <{"static_offsets" = array<i64: 4>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<72xf64>) -> memref<64xf64, strided<[1], offset: 4>>
-// CHECK-NEXT:      %117 = "memref.subview"(%114) <{"static_offsets" = array<i64: 4>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<72xf64>) -> memref<64xf64, strided<[1], offset: 4>>
-// CHECK-NEXT:      %118 = memref.alloc() : memref<64xf64>
-// CHECK-NEXT:      %119 = "memref.subview"(%118) <{"static_offsets" = array<i64: -1>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<64xf64>) -> memref<64xf64, strided<[1], offset: -1>>
+// CHECK-NEXT:    func.func @stencil_buffer(%114 : memref<72xf64>, %115 : memref<72xf64>) {
+// CHECK-NEXT:      %116 = memref.alloc() : memref<64xf64>
+// CHECK-NEXT:      %117 = "memref.subview"(%116) <{"static_offsets" = array<i64: -1>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<64xf64>) -> memref<64xf64, strided<[1], offset: -1>>
+// CHECK-NEXT:      %118 = "memref.subview"(%115) <{"static_offsets" = array<i64: 4>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<72xf64>) -> memref<64xf64, strided<[1], offset: 4>>
+// CHECK-NEXT:      %119 = "memref.subview"(%114) <{"static_offsets" = array<i64: 4>, "static_sizes" = array<i64: 64>, "static_strides" = array<i64: 1>, "operandSegmentSizes" = array<i32: 1, 0, 0, 0>}> : (memref<72xf64>) -> memref<64xf64, strided<[1], offset: 4>>
```

Comment:
The `stencil.buffer` lowering refactor reordered those.
Those are only subviews ("compile-time" ops, they get constant folded anyway) and allocations though, so nothing to worry about AFAIK :slightly_smiling_face: